package.json

```diff
@@ -34,6 +34,7 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "@babel/standalone": "7.0.0-beta.32",
+    "@std/esm": "^0.21.1",
```

Comment:
We use exact dependencies.

Reply:
> Thanks for the pull request. There's one thing I'm wondering about though: Does @std/esm count as a web developer tool? And does it even make sense to consider outside of Node?

Not outside Node no. It's Node for sure. Is there a more appropriate suite?

Reply:
The suite description states
> This is a benchmark suite designed to measure the JavaScript related workloads commonly used by Web Developers nowadays, for example the core workloads in popular tools like Babel or TypeScript.

Lodash is on more than 2 million websites and impacts over 150,000 npm packages (many of those used by web devs). Seems like a good fit.

Reply:
Yeah, good point. I do think we should have a `lodash` benchmark in there for sure. `lodash` is one of the most popular `npm` (frontend) dependencies (if not the most popular).

As for `@std/esm`, we should probably create a Node tooling benchmark for Node only/mostly packages. WDYT?

Reply:
Do you have a good `lodash` benchmark suite already?

Reply:
> Do you have a good lodash benchmark suite already?

I have a pretty robust one. It allows swapping out different versions of Lodash/Underscore. It's a long running benchmark though designed to ensure the things Lodash is good at stay good and the things its bad at stay reasonable.

> As for @std/esm, we should probably create a Node tooling benchmark for Node only/mostly packages. WDYT

That's cool too though so testing the Node flavors of Babel, Webpack, etc.?

Reply:
I'm very interested in the `lodash` benchmark. As long as the runtime is roughly comparable to the other tests it should be fine, and if not we could reduce the iteration count a bit I guess.

> That's cool too though so testing the Node flavors of Babel, Webpack, etc.?

I don't think it makes sense to have webpack there. This is definitely something that should be owned and operated by the [benchmarking WG](https://benchmarking.nodejs.org/). Maybe you can join the call on Monday to discuss the idea?

Reply:
> As long as the runtime is roughly comparable to the other tests it should be fine, and if not we could reduce the iteration count a bit I guess.

It'd need some tweaking it's a [pretty massive suite](https://github.com/lodash/lodash/blob/4.17.5/perf/perf.js).

> This is definitely something that should be owned and operated by the benchmarking WG. Maybe you can join the call on Monday to discuss the idea?

Sure thing! You can DM details if you'd like.

Reply:
> It'd need some tweaking it's a pretty massive suite.

It also seems to mess with the global state of benchmark.js, which needs to be fixed.

> Sure thing! You can DM details if you'd like.

Added note to https://github.com/nodejs/benchmarking/issues/198 for monday's meeting.